ticket.php

```diff
@@ -952,6 +972,8 @@
 
         require_once "ticket_merge_modal.php";
 
+        require_once "ticket_add_product_modal.php";
```

Comment:
Same here re config_module_enable_accounting, can we drop this inside the if statement below?